docs/guides/scripts.md

```diff
@@ -127,11 +127,20 @@ Multiple dependencies can be requested by repeating with `--with` option.
 Note that if `uv run` is used in a _project_, these dependencies will be included _in addition_ to
 the project's dependencies. To opt-out of this behavior, use the `--no-project` flag.
 
-## Declaring script dependencies
+## Creating a python script
```

Comment:
We stylize as `Python` (here, in the body, etc.)

Reply:
Fixed, won't happen again